src/discord/functions.ts

```
@@ -31,49 +31,59 @@ export const Roles = (message: Message, $embed: MessageEmbed) => {
         });
       }).catch(() => { });
   });
-  $embed.setDescription(`Roles last updated on  ${new Date().toString()}`);
+  embed.setDescription(`Roles last updated on  ${new Date().toString()}`);
 };
 
-export const roleremove = (message: Message, commandString: string[], $embed: MessageEmbed) => {
+export const roleremove = async (message: Message, commandString: string[], embed: MessageEmbed) => {
   if (commandString.length !== 2) {
-    $embed.addFields(commands.find((command) => command.name === 'roleremove').fields);
+    embed.addFields(commands.find((command) => command.name === 'roleremove').fields);
     return;
   }
   if (message.member === null) {
-    $embed.setDescription('Cannot remove role. Make sure you are messaging me in any ACM Cyber channel and not in DM.');
+    embed.setDescription('Cannot remove role. Make sure you are messaging me in any ACM Cyber channel and not in DM.');
     return;
```

Instead of returning, just throw an error with these messages in the body.